main.go

```diff
@@ -39,13 +39,14 @@ import (
 
 type contextKey string
 
-const contextKeyError = contextKey("error")
-
-const cloudPlatformScope = "https://www.googleapis.com/auth/cloud-platform"
-
-const Version = "0.1.0"
-const OSArch = runtime.GOOS + "/" + runtime.GOARCH
-const UserAgent = "cloud-run-proxy/" + Version + " (" + OSArch + ")"
+const (
+	contextKeyError    = contextKey("error")
+	cloudPlatformScope = "https://www.googleapis.com/auth/cloud-platform"
+	Version            = "0.1.0"
+	OSArch             = runtime.GOOS + "/" + runtime.GOARCH
+	UserAgent          = "cloud-run-proxy/" + Version + " (" + OSArch + ")"
+	ADCHintMessage     = "Try running `gcloud auth login --update-adc` first then restart the proxy."
```

Comment:
I like this. I would want to preface this with "if you're trying to authenticate using gcloud, try running ..."

Reply:
done